perf/benchmark.jl

```diff
@@ -24,18 +24,18 @@ function benchmark_driver!(f, x...; f_displayname=string(f))
     GC.gc()
 
     print("  Run TapedTask: ")
-    x = (x[1:end-1]..., produce);
+    x = (x[1:end-1]..., produce)
     # show the number of produce calls inside `f`
-    f_task = (f, x; verbose=false) -> begin 
-        tt = TapedTask(f, x...); 
+    f_task = (f, x; verbose=false) -> begin
```

Comment:
Could also use a non-anonymous function here:
```suggestion
    function f_task(f, x; verbose=false)
```